README.md

```diff
@@ -142,6 +142,35 @@ Some CSS rules in the Design System, such as the universal focus styling and the
 </div>
 ```
 
+4. Import the `DSProvider` for Chakra-based components
+
+As we start to integrate and compose DS components from Chakra components, we need to get the styles for those components. In order to do so, we need to wrap all the DS components that internally use with Chakra with a simple provider component.
+
+For now, only a few components will need this provider but as we migrate more components to use a similar pattern of styling, the provider will prove to be useful.
+
+You will want to import the `DSProvider` at the root of your application so that:
```

Comment:
From a consumer's point of view, it won't be clear what needs `DSProvider` and what doesn't - should we make a breaking change bump here and advise everyone to import it going forward?

Reply:
We aren't 1.0 yet so almost every version bump is a breaking change (I've noticed a lot already in the LCA since that hasn't been updated in a while).

I do agree it's not clear what components need it and what components don't. Eventually, all components will need it and all apps will have to use the `DSProvider`. We can make that a migration process or just say from this point on that the apps need (when they might actually not). I prefer to make it explicit so app consumers understand why they need the provider component and for what components. This means we'll have to be more explicit but that's okay.

Reply:
I agree.  The statement should apply to all consuming apps, making it easier for devs to digest and then move forward.